### DISCUSSION TITLE
Anvil not running.

### DISCUSSION THREAD

**Discussion by Cursedisnuts** [Cyfrin/foundry-full-course-cu#2244] [Q&A] [answered]
```
anvil
usage: anvil [-h] [--version]  ...

Generates project structures from Jinja templates

options:
  --version   show version information
  -h, --help  show this help message and exit

Available subcommands:
  
    init      initialize structure from a template
    version   show version information

For subcomamnd help, run `anvil <subcommand> -h`
```
hello I am trying to run anvil but it isn't working it keeps showing this same thing can someone kindly help me resolve this?.

**Comment by Cursedisnuts** [comment, marked as answer]
It seems that the issue occurs when there are multiple entries in the PATH. I somehow managed to solve it on my own.

To fix it, you need to run the bash command  ```nano ~/.bashrc```  Then, scroll to the very bottom, where you'll see multiple entries like this ( feel free to copy this example and paste it into ChatGPT for clarity):

```
export PATH="$PATH:/home/yourusername/.foundry/bin"
export PATH="$PATH:/home/yourusername/.local/bin"
export PATH="$PATH:/home/yourusername/.foundry/bin"
export PATH="$PATH:/home/yourusername/.local/bin"


```
You need to delete the duplicate entries and keep only the necessary ones, like this:

```
export PATH="$PATH:/home/yourusername/.foundry/bin"
export PATH="$PATH:/home/yourusername/.local/bin"

```
Note: I’m not the best at explaining, so if you copy the entries and paste them into ChatGPT, it should help you figure out what to keep and what to remove. After making the changes, press Ctrl+O to save, then follow the prompts and hit Enter. You should now be directed back to your terminal, and if you run Anvil, it should work correctly.

It might be a bit confusing, but keep up the good work and keep getting froggy! 😊

**Reply by EngrPips** [reply]
Thanks for taking the time to explain the solution. It means a lot to us.
